js/bundledPlugins/previouslyUsedKeywords.js

```diff
@@ -0,0 +1,92 @@
+var AssessmentResult = require( "../values/AssessmentResult.js" );
+var isUndefined = require( "lodash/isUndefined" );
+
+/**
+ * @param {object} app The app
+ * @param {object} args An arguments object with usedKeywords, searchUrl, postUrl,
+ * @param {object} i18n The i18n object used for translations
+ * @constructor
+ */
+var PreviouslyUsedKeyword = function( app, args, i18n ) {
+	this.app = app;
+	this.usedKeywords = args.usedKeywords;
+	this.searchUrl = args.searchUrl;
+	this.postUrl = args.postUrl;
+	this.i18n = i18n;
+};
+
+/**
+ * Registers the assessment with the assessor.
+ */
+PreviouslyUsedKeyword.prototype.registerPlugin = function() {
+	this.app.registerAssessment( "usedKeywords", this.assess, "previouslyUsedKeywords" );
+};
+
+/**
+ * Replaces the usedKeywords
+ * @param {object} usedKeywords An object with keywords and ids where they are used.
+ */
+PreviouslyUsedKeyword.prototype.updateKeywordUsage = function( usedKeywords ) {
+	this.usedKeywords = usedKeywords;
+};
+
+/**
+ * Scores the previously used keyword assessment based on the count.
+ * @param {int} count The number of usages found of the keyword.
+ * @param {int} id The ID of a post.
+ * @param {string} keyword The keyword.
+ * @returns {object} the scoreobject with text and score.
+ */
+PreviouslyUsedKeyword.prototype.scoreAssessment = function( count, id, keyword ) {
+	if( count === 0 ) {
+		return {
+			text: this.i18n.dgettext( "js-text-analysis", "You've never used this focus keyword before, very good." ),
+			score: 9
+		};
+	}
+	if( count === 1 ) {
+		var url = "<a href='" + this.postUrl.replace( "{id}", id ) + "'>";
```

Comment:
Possibly extract this replacement action and https://github.com/Yoast/YoastSEO.js/pull/395/files#diff-9da8cedbf92f047bb549025aca5957aeR57. This would also make the defining of the url variable redundant and would lead to cleaner, more readable code.